build-tools/scripts/jdk.mk

```diff
@@ -91,20 +91,23 @@ endif   # Darwin
 ifeq ($(OS),Linux)
 
 # This is for Ubuntu and derivatives (possibly Debian too)
-_LINUX_JAVA_INCLUDE_DIRS          = /usr/lib/jvm/default-java/include/
+_DEFAULT_LINUX_JAVA_INCLUDE_DIRS  = /usr/lib/jvm/default-java/include/
 _LINUX_JAVA_FALLBACK_DIRS         = /usr/lib/jvm/java*
 _LINUX_JAVA_JNI_INCLUDE_DIR       = include
 _LINUX_JAVA_JNI_OS_INCLUDE_DIR    = $(DESKTOP_JAVA_JNI_INCLUDE_DIR)/linux
```

Comment:
Let's get rid of this definition here. `_LINUX_JAVA_JNI_OS_INCLUDE_DIR` isn't used and `DESKTOP_JAVA_JNI_INCLUDE_DIR` isn't set. The whole line is noise